themes/frontierline/includes/nav-util.php

```diff
@@ -6,7 +6,7 @@
 
 <nav id="nav-util" class="can-stick <?php if (get_theme_mod('frontierline_category_drawer') === '1') : ?>has-categories<?php endif; ?>">
   <ul class="content">
-    <li class="nav-util-sidebar"><a href="#sidebar" aria-controls="sidebar" id="toggle-sidebar"><?php _e('Explore', 'frontierline'); ?></a></li>
+    <li class="nav-util-sidebar"><?php if (is_active_sidebar('sidebar')) : ?><a href="#sidebar" aria-controls="sidebar" id="toggle-sidebar"><?php _e('Explore', 'frontierline'); ?></a><?php endif; ?></li>
```

Comment:
We should wrap the condition around the entire `li` so we're not left with an empty list item:

```
<?php if (is_active_sidebar('sidebar')) : ?>
  <li class="nav-util-sidebar"><a href="#sidebar" aria-controls="sidebar" id="toggle-sidebar"><?php _e('Explore', 'frontierline'); ?></a></li>
<?php endif; ?>

Reply:
Good point, I will implement that.

Note for myself: To not break the layout by removing of the `<li>` with `width: 33%`, some CSS need to be added.
```css
#nav-util .content {
  justify-content: flex-end;
}
```

Reply:
Well, it may be a little more complex, given the fact the categories item can be hidden to. It should be rather `space-evenly` or something.